curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/delete-properties-from-a-javascript-object.english.md

```diff
@@ -10,6 +10,33 @@ forumTopicId: 17560
 <section id='description'>
 We can also delete properties from objects like this:
 <code>delete ourDog.bark;</code>
+
+Example:
+
+```js
+var ourDog = {
+  "name": "Camper",
+  "legs": 4,
+  "tails": 1,
+  "friends": ["everything!"],
+  "bark": "bow-wow"
+};
+
+delete ourDog.bark;
+```
+
+After the last line shown above, <code>ourDog</code> looks like:
+
+```js
+{
+  "name": "Camper",
+  "legs": 4,
+  "tails": 1,
+  "friends": ["everything!"]
+}
+*/
```

Comment:
```suggestion
*/ <- Is this line a typo?
```

Reply:
Yes it is a typo.  Can you make a suggestion to remove it and I will commit it?